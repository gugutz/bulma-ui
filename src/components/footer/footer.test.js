import React from 'react'
import renderer from 'react-test-renderer'
import Footer from './footer'

describe('<Footer /> component', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<Footer>text</Footer>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
