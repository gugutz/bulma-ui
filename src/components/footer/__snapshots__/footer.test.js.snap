// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> component should render correctly 1`] = `
<div
  className="footer"
>
  text
</div>
`;
